docker/monitoring/prometheus.yaml

```diff
@@ -11,4 +11,7 @@ scrape_configs:
       - targets: [ 'tempo:3200' ]
   - job_name: 'booster-http'
     static_configs:
-      - targets: [ 'booster-http:7777' ]
\ No newline at end of file
+      - targets: [ 'booster-http:7777' ]
+  - job_name: 'lotus-miner'
+    static_configs:
+      - targets: [ 'host.docker.internal:8787/debug/metrics' ]
```

Comment:
`lotus-miner` is not necessarily on the local host, and not necessarily located on `:8787`